pkg/authenticator/k8s/config.go

```diff
@@ -92,10 +92,6 @@ func (config *Config) LoadConfig(settings map[string]string) {
 	}
 }
 
-func (config *Config) GetAuthenticationType() string {
```

Comment:
what is the purpose of deleting this getter?

Reply:
There is no use for it. There is const in each package for the authenticator. In addition we can check the type of the object.